10/19/2016, 1:40:25 PM - info: Connecting to MongoDB
10/19/2016, 1:40:25 PM - info: Connecting to RabbitMQ
10/19/2016, 1:40:25 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 1:40:25 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 1:40:25 PM - info: Express server listening on port 9010
10/19/2016, 1:40:25 PM - error:  Error: connect ECONNREFUSED 127.0.0.1:5672
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
10/19/2016, 1:40:25 PM - info: MongoDB Connected!
10/19/2016, 1:40:25 PM - info: getIdArrayFromObjects in attribute service called
10/19/2016, 1:40:26 PM - info: Connecting to RabbitMQ
10/19/2016, 1:40:26 PM - error:  Error: connect ECONNREFUSED 127.0.0.1:5672
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
10/19/2016, 1:40:27 PM - info: Connecting to RabbitMQ
10/19/2016, 1:40:27 PM - error:  Error: connect ECONNREFUSED 127.0.0.1:5672
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
10/19/2016, 1:40:28 PM - info: Connecting to RabbitMQ
10/19/2016, 1:40:28 PM - error:  Error: connect ECONNREFUSED 127.0.0.1:5672
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
10/19/2016, 1:40:50 PM - info: Connecting to MongoDB
10/19/2016, 1:40:50 PM - info: Connecting to RabbitMQ
10/19/2016, 1:40:50 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 1:40:50 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 1:40:50 PM - info: Express server listening on port 9010
10/19/2016, 1:40:50 PM - error:  Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
10/19/2016, 1:40:50 PM - error: call attribute service in attribute service returned error for categoryId : 1000 { Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 9000 }
10/19/2016, 1:40:51 PM - info: MongoDB Connected!
10/19/2016, 1:40:51 PM - info: RabbitMQ connected
10/19/2016, 3:03:39 PM - info: Received SIGINT
10/19/2016, 3:03:39 PM - info: Cleanup called
10/19/2016, 3:03:39 PM - info: MongoDB cleanup called
10/19/2016, 3:03:39 PM - info: Rabbitmq cleanup called
10/19/2016, 3:03:39 PM - info: MongoDB connection closed
10/19/2016, 3:03:39 PM - info: RabbitMQ connection closed
10/19/2016, 4:07:02 PM - error: error
10/19/2016, 4:07:02 PM - info: getAttributes in attribute service called 
10/19/2016, 4:07:02 PM - error: error
10/19/2016, 4:07:02 PM - error: error
10/19/2016, 4:07:02 PM - error: error
10/19/2016, 4:07:02 PM - error: error
10/19/2016, 4:07:02 PM - error: error
10/19/2016, 4:07:02 PM - error: error
10/19/2016, 4:07:02 PM - error: error
10/19/2016, 4:07:02 PM - error: error
10/19/2016, 4:07:02 PM - error: error
10/19/2016, 4:07:02 PM - error: error
10/19/2016, 4:07:02 PM - error:  NotFoundError: not found
    at Context.<anonymous> (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/test/api/jobseekerapi-post-test.js:56:91)
    at callFnAsync (/usr/local/lib/node_modules/mocha/lib/runnable.js:309:8)
    at Test.Runnable.run (/usr/local/lib/node_modules/mocha/lib/runnable.js:264:7)
    at Runner.runTest (/usr/local/lib/node_modules/mocha/lib/runner.js:419:10)
    at /usr/local/lib/node_modules/mocha/lib/runner.js:526:12
    at next (/usr/local/lib/node_modules/mocha/lib/runner.js:340:14)
    at /usr/local/lib/node_modules/mocha/lib/runner.js:350:7
    at next (/usr/local/lib/node_modules/mocha/lib/runner.js:283:14)
    at /usr/local/lib/node_modules/mocha/lib/runner.js:313:7
    at done (/usr/local/lib/node_modules/mocha/lib/runnable.js:250:5)
    at callFn (/usr/local/lib/node_modules/mocha/lib/runnable.js:304:7)
    at Hook.Runnable.run (/usr/local/lib/node_modules/mocha/lib/runnable.js:282:7)
    at next (/usr/local/lib/node_modules/mocha/lib/runner.js:295:10)
    at Immediate._onImmediate (/usr/local/lib/node_modules/mocha/lib/runner.js:318:5)
    at tryOnImmediate (timers.js:543:15)
    at processImmediate [as _immediateCallback] (timers.js:523:5)
10/19/2016, 4:07:02 PM - error: error
10/19/2016, 4:07:03 PM - error: findByJobSeekerId in jobseekerepository returned error Could not connect
10/19/2016, 4:07:03 PM - error: findById in jobseekerepository returned error Could not connect
10/19/2016, 4:07:03 PM - info: Save to DB called
10/19/2016, 4:07:03 PM - info: saveToDB:27.374634999998307
10/19/2016, 4:07:03 PM - info: Save to DB called
10/19/2016, 4:07:03 PM - info: saveToDB:12.132596000001286
10/19/2016, 4:07:03 PM - info: Save to DB called
10/19/2016, 4:07:03 PM - info: saveToDB:25.993505999998888
10/19/2016, 4:07:03 PM - info: getCategoryAttributeList in attribute service called for categoryId : 2
10/19/2016, 4:07:03 PM - info: callAttributeService in attribute service called for categoryId : 2
10/19/2016, 4:07:03 PM - info: getIdArrayFromObjects in attribute service called
10/19/2016, 4:07:03 PM - info: getCategoryAttributeList in attribute service called for categoryId : 2
10/19/2016, 4:07:03 PM - info: callAttributeService in attribute service called for categoryId : 2
10/19/2016, 4:07:03 PM - error: call attribute service in attribute service returned error for categoryId : 2 Could not connect
10/19/2016, 4:07:03 PM - info: getCategoryAttributeList in attribute service called for categoryId : 2
10/19/2016, 4:07:03 PM - info: callAttributeService in attribute service called for categoryId : 2
10/19/2016, 4:07:03 PM - info: getAttributes in attribute service called 
10/19/2016, 4:07:04 PM - info: getAttributes in attribute service called 
10/19/2016, 4:07:04 PM - info: getAttributes in attribute service called 
10/19/2016, 4:07:04 PM - info: getCategoryAttributeList in attribute service called for categoryId : 2
10/19/2016, 4:07:04 PM - info: callAttributeService in attribute service called for categoryId : 2
10/19/2016, 4:07:04 PM - info: getIdArrayFromObjects in attribute service called
10/19/2016, 4:07:04 PM - info: getInterviewsForJobseeker in interview service called 
10/19/2016, 4:07:04 PM - info: getInterviewsForJobseeker in interview service called 
10/19/2016, 4:07:04 PM - error: getInterviewsForJobseeker passthrough returned error for jobSeekerId : 5742908688b10bdb1bcd1a39 Could not connect
10/19/2016, 4:07:04 PM - info: create in jobapplication service called 
10/19/2016, 4:07:04 PM - info: amendRequestBodyAndSendRequest in jobapplication called for http://core-nancy.babajob.com/v2/jobApplications/
10/19/2016, 4:07:04 PM - info: create in jobapplication service called 
10/19/2016, 4:07:04 PM - info: amendRequestBodyAndSendRequest in jobapplication called for http://core-nancy.babajob.com/v2/jobApplications/
10/19/2016, 4:07:04 PM - info: create in jobapplication service called 
10/19/2016, 4:07:04 PM - info: amendRequestBodyAndSendRequest in jobapplication called for http://core-nancy.babajob.com/v2/jobApplications/
10/19/2016, 4:07:04 PM - info: update in jobapplication service called 
10/19/2016, 4:07:04 PM - info: amendRequestBodyAndSendRequest in jobapplication called for http://core-nancy.babajob.com/v2/jobApplications/3245
10/19/2016, 4:07:04 PM - info: update in jobapplication service called 
10/19/2016, 4:07:04 PM - info: amendRequestBodyAndSendRequest in jobapplication called for http://core-nancy.babajob.com/v2/jobApplications/3245
10/19/2016, 4:07:04 PM - info: update in jobapplication service called 
10/19/2016, 4:07:04 PM - info: amendRequestBodyAndSendRequest in jobapplication called for http://core-nancy.babajob.com/v2/jobApplications/3245
10/19/2016, 4:07:04 PM - info: getApplication in jobapplication service called 
10/19/2016, 4:07:04 PM - info: getApplication in jobapplication service called 
10/19/2016, 4:07:04 PM - info: getApplication in jobapplication service called 
10/19/2016, 4:07:04 PM - info: getApplicationsForJobseeker in jobapplication service called 
10/19/2016, 4:07:04 PM - info: getApplicationsForJobseeker in jobapplication service called 
10/19/2016, 4:07:04 PM - error: getApplicationsForJobseeker passthrough returned error for jobSeekerId : agfhjk5678 Could not connect
10/19/2016, 4:07:04 PM - info: getApplicationsForJobseeker in jobapplication service called 
10/19/2016, 4:07:04 PM - error: getApplicationsForJobseeker passthrough returned error for jobSeekerId : agfhjk5678 statusCode=404, body=Not in system
10/19/2016, 4:07:04 PM - info: getQuestionsForApplication in jobapplication service called 
10/19/2016, 4:07:04 PM - info: getQuestionsForApplication in jobapplication service called 
10/19/2016, 4:07:04 PM - error: getApplicationsForJobseeker passthrough returned error for application id : 3245 Could not connect
10/19/2016, 4:07:04 PM - info: getQuestionsForApplication in jobapplication service called 
10/19/2016, 4:07:04 PM - error: getApplicationsForJobseeker passthrough returned error for application id : 3245 statusCode=404, body=Not in system
10/19/2016, 4:07:04 PM - info: getSeekerFromDb in jobseekerquestionsservice  called for jobSeekerId : undefined
10/19/2016, 4:07:04 PM - info: getAttributesForCategory in jobseekerquestionsservice  called for jobSeekerId : undefined
10/19/2016, 4:07:04 PM - info: getAttributeUnion in jobseekerquestionsservice  called 
10/19/2016, 4:07:04 PM - info: getQuestionsForAttributes in jobseekerquestionsservice  called 
10/19/2016, 4:07:04 PM - info: fillAnswersForAnsweredQuestions in jobseekerquestionsservice  called 
10/19/2016, 4:07:04 PM - info: getSeekerFromDb in jobseekerquestionsservice  called for jobSeekerId : 123
10/19/2016, 4:07:04 PM - info: getSeekerFromDb in jobseekerquestionsservice  called for jobSeekerId : 123
10/19/2016, 4:07:04 PM - info: getAttributesForCategory in jobseekerquestionsservice  called for jobSeekerId : 123
10/19/2016, 4:07:04 PM - info: getSeekerFromDb in jobseekerquestionsservice  called for jobSeekerId : 123
10/19/2016, 4:07:04 PM - info: getAttributesForCategory in jobseekerquestionsservice  called for jobSeekerId : 123
10/19/2016, 4:07:04 PM - info: getAttributeUnion in jobseekerquestionsservice  called 
10/19/2016, 4:07:04 PM - info: getQuestionsForAttributes in jobseekerquestionsservice  called 
10/19/2016, 4:07:04 PM - info: getSeekerAttributesAndCategoryFromDb in jobseekerquestionsservice  called for jobSeekerId : undefined
10/19/2016, 4:07:04 PM - info: getAttributesForCategory in jobseekerquestionsservice  called for jobSeekerId : undefined
10/19/2016, 4:07:04 PM - info: getAttributeDifference in jobseekerquestionsservice  called 
10/19/2016, 4:07:04 PM - info: getQuestionsForAttributes in jobseekerquestionsservice  called 
10/19/2016, 4:07:04 PM - info: getSeekerAttributesAndCategoryFromDb in jobseekerquestionsservice  called for jobSeekerId : 123
10/19/2016, 4:07:04 PM - error: findAttributesAndJobCategoryById from jobseekerrepository returned error NotFoundError: no fields found
    at Context.<anonymous> (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/test/services/jobseekerquestionservice-getunansweredquestions-test.js:102:94)
    at callFnAsync (/usr/local/lib/node_modules/mocha/lib/runnable.js:309:8)
    at Test.Runnable.run (/usr/local/lib/node_modules/mocha/lib/runnable.js:264:7)
    at Runner.runTest (/usr/local/lib/node_modules/mocha/lib/runner.js:419:10)
    at /usr/local/lib/node_modules/mocha/lib/runner.js:526:12
    at next (/usr/local/lib/node_modules/mocha/lib/runner.js:340:14)
    at /usr/local/lib/node_modules/mocha/lib/runner.js:350:7
    at next (/usr/local/lib/node_modules/mocha/lib/runner.js:283:14)
    at /usr/local/lib/node_modules/mocha/lib/runner.js:313:7
    at done (/usr/local/lib/node_modules/mocha/lib/runnable.js:250:5)
    at callFn (/usr/local/lib/node_modules/mocha/lib/runnable.js:304:7)
    at Hook.Runnable.run (/usr/local/lib/node_modules/mocha/lib/runnable.js:282:7)
    at next (/usr/local/lib/node_modules/mocha/lib/runner.js:295:10)
    at Immediate._onImmediate (/usr/local/lib/node_modules/mocha/lib/runner.js:318:5)
    at tryOnImmediate (timers.js:543:15)
    at processImmediate [as _immediateCallback] (timers.js:523:5)
10/19/2016, 4:07:04 PM - info: getSeekerAttributesAndCategoryFromDb in jobseekerquestionsservice  called for jobSeekerId : 123
10/19/2016, 4:07:04 PM - info: getAttributesForCategory in jobseekerquestionsservice  called for jobSeekerId : 123
10/19/2016, 4:07:04 PM - info: getSeekerAttributesAndCategoryFromDb in jobseekerquestionsservice  called for jobSeekerId : 123
10/19/2016, 4:07:04 PM - info: getAttributesForCategory in jobseekerquestionsservice  called for jobSeekerId : 123
10/19/2016, 4:07:04 PM - info: getAttributeDifference in jobseekerquestionsservice  called 
10/19/2016, 4:07:04 PM - info: getQuestionsForAttributes in jobseekerquestionsservice  called 
10/19/2016, 4:07:04 PM - info: validateSchema in jobseekerservice called 
10/19/2016, 4:07:04 PM - info: registeruser in jobseekerservice called 
10/19/2016, 4:07:04 PM - info: saveToDB in jobseekerservice called for 12345
10/19/2016, 4:07:04 PM - info: inSave in jobseekerservice called 
10/19/2016, 4:07:04 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 4:07:04 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 4:07:04 PM - info: validateSchema in jobseekerservice called 
10/19/2016, 4:07:04 PM - info: registeruser in jobseekerservice called 
10/19/2016, 4:07:04 PM - info: saveToDB in jobseekerservice called for 12345
10/19/2016, 4:07:04 PM - info: inSave in jobseekerservice called 
10/19/2016, 4:07:04 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 4:07:04 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 4:07:04 PM - info: validateSchema in jobseekerservice called 
10/19/2016, 4:07:04 PM - info: validateSchema in jobseekerservice called 
10/19/2016, 4:07:04 PM - info: registeruser in jobseekerservice called 
10/19/2016, 4:07:04 PM - error: register user service returned error statusCode=403, body=This mobile number is taken
10/19/2016, 4:07:04 PM - info: validateSchema in jobseekerservice called 
10/19/2016, 4:07:04 PM - info: registeruser in jobseekerservice called 
10/19/2016, 4:07:04 PM - info: saveToDB in jobseekerservice called for 12345
10/19/2016, 4:07:04 PM - info: deleteUser in userservice called for userid 
10/19/2016, 4:07:04 PM - info: validateSchema in jobseekerservice called 
10/19/2016, 4:07:04 PM - info: registeruser in jobseekerservice called 
10/19/2016, 4:07:04 PM - info: saveToDB in jobseekerservice called for 12345
10/19/2016, 4:07:04 PM - info: inSave in jobseekerservice called 
10/19/2016, 4:07:04 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 4:07:04 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 4:07:04 PM - info: deleteUser in jobseekerservice called
10/19/2016, 4:07:04 PM - info: getJobSeeker in jobseekerservice  called for 1234567890
10/19/2016, 4:07:04 PM - info: updateToDB in jobseekerservice called 
10/19/2016, 4:07:04 PM - info: deleteUser in jobseekerservice called
10/19/2016, 4:07:04 PM - info: getJobSeeker in jobseekerservice  called for 1234567890
10/19/2016, 4:07:04 PM - info: deleteUser in jobseekerservice called
10/19/2016, 4:07:04 PM - info: getJobSeeker in jobseekerservice  called for 1234567890
10/19/2016, 4:07:04 PM - info: updateToDB in jobseekerservice called 
10/19/2016, 4:07:04 PM - info: deleteUser in jobseekerservice called
10/19/2016, 4:07:04 PM - info: getJobSeeker in jobseekerservice  called for 1234567890
10/19/2016, 4:07:04 PM - info: getJobSeeker in jobseekerservice  called for 5742908688b10bdb1bcd1a39
10/19/2016, 4:07:04 PM - info: getJobSeeker in jobseekerservice  called for 5742908688b10bdb1bcd1a39
10/19/2016, 4:07:04 PM - info: getJobSeekerByUserId in jobseekerservice called for 12345
10/19/2016, 4:07:04 PM - info: getJobSeekerByUserId in jobseekerservice called for 12345
10/19/2016, 4:07:04 PM - error: findByJobSeekerId returned error Not found
10/19/2016, 4:07:04 PM - info: validateSchema in jobseekerservice called 
10/19/2016, 4:07:04 PM - info: validateSchema in jobseekerservice called 
10/19/2016, 4:07:04 PM - info: updateUser in jobseekerservice called for 12rfghjk
10/19/2016, 4:07:04 PM - info: getJobSeeker in jobseekerservice  called for 12rfghjk
10/19/2016, 4:07:04 PM - info: updateToDB in jobseekerservice called 
10/19/2016, 4:07:04 PM - info: inSave in jobseekerservice called 
10/19/2016, 4:07:04 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 4:07:04 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 4:07:04 PM - info: validateSchema in jobseekerservice called 
10/19/2016, 4:07:04 PM - info: updateUser in jobseekerservice called for 12rfghjk
10/19/2016, 4:07:04 PM - info: getJobSeeker in jobseekerservice  called for 12rfghjk
10/19/2016, 4:07:05 PM - info: validateSchema in jobseekerservice called 
10/19/2016, 4:07:05 PM - info: updateUser in jobseekerservice called for 12rfghjk
10/19/2016, 4:07:05 PM - info: getJobSeeker in jobseekerservice  called for 12rfghjk
10/19/2016, 4:07:05 PM - info: getIdArrayFromObjects in attribute service called
10/19/2016, 4:07:05 PM - info: getIdArrayFromObjects in attribute service called
10/19/2016, 4:07:05 PM - info: getIdArrayFromObjects in attribute service called
10/19/2016, 4:07:05 PM - info: validateSchema in jobseekerservice called 
10/19/2016, 4:07:05 PM - info: updateUser in jobseekerservice called for 12rfghjk
10/19/2016, 4:07:05 PM - info: getJobSeeker in jobseekerservice  called for 12rfghjk
10/19/2016, 4:07:05 PM - info: updateToDB in jobseekerservice called 
10/19/2016, 4:07:05 PM - error: failed to delete user from sql when mongodb failed : 12345 statusCode=401, body=undefined
10/19/2016, 4:07:05 PM - info: validateSchema in jobseekerservice called 
10/19/2016, 4:07:05 PM - info: validateSchema in jobseekerservice called 
10/19/2016, 4:07:05 PM - info: updateUser in jobseekerservice called for 1234567890
10/19/2016, 4:07:05 PM - info: getJobSeeker in jobseekerservice  called for 1234567890
10/19/2016, 4:07:05 PM - info: updateToDB in jobseekerservice called 
10/19/2016, 4:07:05 PM - info: inSave in jobseekerservice called 
10/19/2016, 4:07:05 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 4:07:05 PM - info: validateSchema in jobseekerservice called 
10/19/2016, 4:07:05 PM - info: updateUser in jobseekerservice called for 1234567890
10/19/2016, 4:07:05 PM - info: getJobSeeker in jobseekerservice  called for 1234567890
10/19/2016, 4:07:05 PM - info: getIdArrayFromObjects in attribute service called
10/19/2016, 4:07:05 PM - info: validateSchema in jobseekerservice called 
10/19/2016, 4:07:05 PM - info: updateUser in jobseekerservice called for 1234567890
10/19/2016, 4:07:05 PM - info: getJobSeeker in jobseekerservice  called for 1234567890
10/19/2016, 4:07:05 PM - info: updateToDB in jobseekerservice called 
10/19/2016, 4:07:05 PM - info: validateSchema in jobseekerservice called 
10/19/2016, 4:07:05 PM - info: updateUser in jobseekerservice called for 1234567890
10/19/2016, 4:07:05 PM - info: getJobSeeker in jobseekerservice  called for 1234567890
10/19/2016, 4:07:05 PM - info: validateSchema in jobseekerservice called 
10/19/2016, 4:07:05 PM - info: updateUser in jobseekerservice called for 1234567890
10/19/2016, 4:07:05 PM - info: getJobSeeker in jobseekerservice  called for 1234567890
10/19/2016, 4:07:05 PM - info: updateToDB in jobseekerservice called 
10/19/2016, 4:07:05 PM - info: inSave in jobseekerservice called 
10/19/2016, 4:07:05 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 4:07:05 PM - info: validateSchema in jobseekerservice called 
10/19/2016, 4:07:05 PM - info: updateUser in jobseekerservice called for 1234567890
10/19/2016, 4:07:05 PM - info: getJobSeeker in jobseekerservice  called for 1234567890
10/19/2016, 4:07:05 PM - info: updateToDB in jobseekerservice called 
10/19/2016, 4:07:05 PM - info: inSave in jobseekerservice called 
10/19/2016, 4:07:05 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 4:07:05 PM - info: getQuestions in questionsservice  called 
10/19/2016, 4:07:05 PM - info: getQuestions in questionsservice  called 
10/19/2016, 4:07:05 PM - info: getQuestions in questionsservice  called 
10/19/2016, 4:07:05 PM - info: deleteUser in userservice called for userid 
10/19/2016, 4:07:05 PM - info: deleteUser in userservice called for userid 
10/19/2016, 4:07:05 PM - error: deleteUser request in userservice returned error for userId :12345 Could not connect
10/19/2016, 4:07:06 PM - info: deleteUser in userservice called for userid 
10/19/2016, 4:07:06 PM - error: deleteUser request in userservice returned error for userId :12345 statusCode=404, body=User does not exist
10/19/2016, 4:07:06 PM - info: registerUser:1.2873959999997169
10/19/2016, 4:07:06 PM - info: registerUser:0.4704430000001594
10/19/2016, 4:07:06 PM - info: registerUser:0.8291790000002948
10/19/2016, 4:07:07 PM - error: update request in userservice returned error for userId :undefined Could not connect
10/19/2016, 4:07:07 PM - error: update request in userservice returned error for userId :undefined statusCode=403, body=This mobile number is already taken
10/19/2016, 4:09:53 PM - info: Connecting to MongoDB
10/19/2016, 4:09:53 PM - info: Connecting to RabbitMQ
10/19/2016, 4:09:53 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 4:09:53 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 4:09:54 PM - info: Express server listening on port 9010
10/19/2016, 4:09:54 PM - info: getIdArrayFromObjects in attribute service called
10/19/2016, 4:09:54 PM - info: RabbitMQ connected
10/19/2016, 4:10:04 PM - info: In Uncaught exception: 
10/19/2016, 4:10:04 PM - error: Error: Uncaught, unspecified "error" event. ([object Object])
    at NativeConnection.emit (events.js:163:17)
    at /Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/mongoose/lib/connection.js:428:17
    at NativeConnection.Connection.error (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/mongoose/lib/connection.js:451:12)
    at /Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/mongoose/lib/connection.js:482:15
    at /Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/mongoose/lib/drivers/node-mongodb-native/connection.js:216:21
    at /Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/mongodb/lib/db.js:229:14
    at .<anonymous> (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/mongodb/lib/replset.js:357:9)
    at g (events.js:286:16)
    at emitOne (events.js:96:13)
    at emit (events.js:188:7)
    at Timeout.<anonymous> (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/mongodb-core/lib/topologies/replset.js:507:14)
    at tryOnTimeout (timers.js:224:11)
    at Timer.listOnTimeout (timers.js:198:5)
10/19/2016, 4:10:04 PM - info: Cleanup called
10/19/2016, 4:10:04 PM - info: MongoDB cleanup called
10/19/2016, 4:10:04 PM - info: MongoDB connection closed
10/19/2016, 4:10:04 PM - info: Rabbitmq cleanup called
10/19/2016, 4:10:04 PM - info: Process exiting: 1
10/19/2016, 4:10:37 PM - info: Connecting to MongoDB
10/19/2016, 4:10:37 PM - info: Connecting to RabbitMQ
10/19/2016, 4:10:37 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 4:10:37 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 4:10:38 PM - info: Express server listening on port 9010
10/19/2016, 4:10:38 PM - info: getIdArrayFromObjects in attribute service called
10/19/2016, 4:10:38 PM - info: RabbitMQ connected
10/19/2016, 4:10:42 PM - info: In Uncaught exception: 
10/19/2016, 4:10:42 PM - error: MongoError: no primary found in replicaset
    at /Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/mongodb-core/lib/topologies/replset.js:631:32
    at .<anonymous> (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/mongodb-core/lib/topologies/replset.js:421:24)
    at g (events.js:286:16)
    at emitOne (events.js:96:13)
    at emit (events.js:188:7)
    at .<anonymous> (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/mongodb-core/lib/topologies/server.js:313:21)
    at emitOne (events.js:96:13)
    at emit (events.js:188:7)
    at .<anonymous> (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/mongodb-core/lib/connection/pool.js:260:12)
    at g (events.js:286:16)
    at emitTwo (events.js:106:13)
    at emit (events.js:191:7)
    at Socket.<anonymous> (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/mongodb-core/lib/connection/connection.js:162:49)
    at Socket.g (events.js:286:16)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
10/19/2016, 4:10:42 PM - info: Cleanup called
10/19/2016, 4:10:42 PM - info: MongoDB cleanup called
10/19/2016, 4:10:42 PM - info: MongoDB connection closed
10/19/2016, 4:10:42 PM - info: Rabbitmq cleanup called
10/19/2016, 4:10:42 PM - info: Process exiting: 1
10/19/2016, 4:11:05 PM - info: Connecting to MongoDB
10/19/2016, 4:11:05 PM - info: Connecting to RabbitMQ
10/19/2016, 4:11:05 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 4:11:05 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 4:11:05 PM - info: Express server listening on port 9010
10/19/2016, 4:11:06 PM - info: getIdArrayFromObjects in attribute service called
10/19/2016, 4:11:06 PM - info: In Uncaught exception: 
10/19/2016, 4:11:38 PM - info: Connecting to MongoDB
10/19/2016, 4:11:38 PM - info: Connecting to RabbitMQ
10/19/2016, 4:11:38 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 4:11:39 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 4:11:39 PM - info: Express server listening on port 9010
10/19/2016, 4:11:39 PM - error:  Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
10/19/2016, 4:11:39 PM - error: call attribute service in attribute service returned error for categoryId : 1000 { Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 9000 }
10/19/2016, 4:11:39 PM - info: MongoDB Connected!
10/19/2016, 4:11:39 PM - info: RabbitMQ connected
10/19/2016, 4:12:07 PM - error: error on decoding token : eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJxYWFwaS5iYWJham9iLmNvbSIsImV4cCI6MTQ5MTcyMTI3NywidG9rZW5JZCI6ImU0NTYxYTJhZmMzMTQwNTliNTdlZDRmZGNmZjI0NDY2IiwidXNlcklkIjoiNDA5OTc5OSIsInVzZXJOYW1lIjoiYXNodXRvc2hhQGJhYmFqb2IuY29tIiwicm9sZSI6IkFETUlOIiwicHJvZmlsZUlkIjoiNDA5OTc5OSIsImNvbnN1bWVyS2V5IjoiMmtzUngzZTZUTElOUEtXOGRlM1VmMURjMjg4ZHhTIiwic2NvcGUiOiJBQ0NFU1MifQ._3ITWGrhaW-P8S8gk0HsP1eEVMOTPKkA4j1cw5K3Ad8, consumer-key:  2ksRx3e6TLINPKW8de3Uf1Dc288dxS, req : /jobseekers/57f73cde5cc101607a49cb89/applications, err :  %s JsonWebTokenError: secret or public key must be provided
    at Object.module.exports [as verify] (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/jsonwebtoken/verify.js:54:17)
    at Object.exports.authenticate.err [as authenticate] (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/src/lib/requestAuthentication.js:31:25)
    at app.use (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/src/app.js:33:25)
    at Layer.handle [as handle_request] (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/express/lib/router/index.js:312:13)
    at /Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/express/lib/router/index.js:280:7
    at Function.process_params (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/express/lib/router/index.js:330:12)
    at next (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/express/lib/router/index.js:271:10)
    at /Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/express-winston/index.js:291:9
    at Layer.handle [as handle_request] (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/express/lib/router/index.js:312:13)
    at /Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/express/lib/router/index.js:280:7
    at Function.process_params (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/express/lib/router/index.js:330:12)
    at next (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/express/lib/router/index.js:271:10)
    at SendStream.error (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/serve-static/index.js:121:7)
    at emitOne (events.js:96:13)
10/19/2016, 4:12:32 PM - info: Received SIGINT
10/19/2016, 4:12:32 PM - info: Cleanup called
10/19/2016, 4:12:32 PM - info: MongoDB cleanup called
10/19/2016, 4:12:32 PM - info: Rabbitmq cleanup called
10/19/2016, 4:12:32 PM - info: MongoDB connection closed
10/19/2016, 4:12:32 PM - info: RabbitMQ connection closed
10/19/2016, 4:12:35 PM - info: Connecting to MongoDB
10/19/2016, 4:12:35 PM - info: Connecting to RabbitMQ
10/19/2016, 4:12:35 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 4:12:35 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 4:12:35 PM - info: Express server listening on port 9010
10/19/2016, 4:12:35 PM - error:  Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
10/19/2016, 4:12:35 PM - error: call attribute service in attribute service returned error for categoryId : 1000 { Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 9000 }
10/19/2016, 4:12:35 PM - info: MongoDB Connected!
10/19/2016, 4:12:36 PM - info: RabbitMQ connected
10/19/2016, 4:12:44 PM - info: getApplicationsForJobseeker in jobapplication service called 
10/19/2016, 4:12:44 PM - info: getRequestWrapper in requestWrapper called
10/19/2016, 4:12:44 PM - error: getApplicationsForJobseeker passthrough returned error for jobSeekerId : 57f73cde5cc101607a49cb89 statusCode=405, message=undefined
10/19/2016, 4:12:44 PM - error:  statusCode=405, message=undefined
10/19/2016, 4:13:32 PM - info: getSeekerAttributesAndCategoryFromDb in jobseekerquestionsservice  called for jobSeekerId : 57f73cde5cc101607a49cb89
10/19/2016, 4:13:32 PM - error:  NotFoundError: Jobseeker with jobSeekerId : 57f73cde5cc101607a49cb89 not found
    at /Users/shubhammakwana/Documents/Workspace/job-seeker-api/src/repository/jobseekerrepository.js:118:16
    at Query.<anonymous> (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/mongoose/lib/model.js:3343:16)
    at /Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/kareem/index.js:259:21
    at /Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/kareem/index.js:127:16
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickDomainCallback (internal/process/next_tick.js:122:9)
10/19/2016, 4:13:32 PM - error: findAttributesAndJobCategoryById from jobseekerrepository returned error NotFoundError: Jobseeker with jobSeekerId : 57f73cde5cc101607a49cb89 not found
    at /Users/shubhammakwana/Documents/Workspace/job-seeker-api/src/repository/jobseekerrepository.js:118:16
    at Query.<anonymous> (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/mongoose/lib/model.js:3343:16)
    at /Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/kareem/index.js:259:21
    at /Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/kareem/index.js:127:16
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickDomainCallback (internal/process/next_tick.js:122:9)
10/19/2016, 4:14:40 PM - info: validateSchema in jobseekerservice called 
10/19/2016, 4:14:41 PM - error: error on request 1476873880970 POST /jobseekers TypeError: validate is not a function
    at /Users/shubhammakwana/Documents/Workspace/job-seeker-api/src/lib/validation.js:23:15
    at schemaLoaded (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/ajv/lib/async.js:196:25)
    at /Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/ajv/lib/async.js:187:13
    at Request._callback (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/src/lib/validation.js:14:4)
    at self.callback (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/request/request.js:187:22)
    at emitOne (events.js:96:13)
    at Request.emit (events.js:188:7)
    at Request.onRequestError (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/request/request.js:813:8)
    at emitOne (events.js:96:13)
    at ClientRequest.emit (events.js:188:7)
    at Socket.socketErrorListener (_http_client.js:306:9)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at emitErrorNT (net.js:1272:8)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickDomainCallback (internal/process/next_tick.js:122:9)
10/19/2016, 4:14:59 PM - info: validateSchema in jobseekerservice called 
10/19/2016, 4:14:59 PM - error: error on request 1476873899031 POST /jobSeekers TypeError: validate is not a function
    at /Users/shubhammakwana/Documents/Workspace/job-seeker-api/src/lib/validation.js:23:15
    at schemaLoaded (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/ajv/lib/async.js:196:25)
    at /Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/ajv/lib/async.js:187:13
    at Request._callback (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/src/lib/validation.js:14:4)
    at self.callback (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/request/request.js:187:22)
    at emitOne (events.js:96:13)
    at Request.emit (events.js:188:7)
    at Request.onRequestError (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/request/request.js:813:8)
    at emitOne (events.js:96:13)
    at ClientRequest.emit (events.js:188:7)
    at Socket.socketErrorListener (_http_client.js:306:9)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at emitErrorNT (net.js:1272:8)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickDomainCallback (internal/process/next_tick.js:122:9)
10/19/2016, 4:16:57 PM - info: getSeekerAttributesAndCategoryFromDb in jobseekerquestionsservice  called for jobSeekerId : 58074ee0e10743268790b228
10/19/2016, 4:17:15 PM - info: Received SIGINT
10/19/2016, 4:17:15 PM - info: Cleanup called
10/19/2016, 4:17:15 PM - info: MongoDB cleanup called
10/19/2016, 4:17:15 PM - info: Rabbitmq cleanup called
10/19/2016, 4:17:15 PM - info: MongoDB connection closed
10/19/2016, 4:17:15 PM - info: RabbitMQ connection closed
10/19/2016, 4:17:35 PM - info: Connecting to MongoDB
10/19/2016, 4:17:35 PM - info: Connecting to RabbitMQ
10/19/2016, 4:17:35 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 4:17:35 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 4:17:43 PM - info: Express server listening on port 9010
10/19/2016, 4:17:43 PM - error:  Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
10/19/2016, 4:17:43 PM - error: call attribute service in attribute service returned error for categoryId : 1000 { Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 9000 }
10/19/2016, 4:17:43 PM - info: MongoDB Connected!
10/19/2016, 4:17:43 PM - info: RabbitMQ connected
10/19/2016, 4:56:14 PM - info: Connecting to MongoDB
10/19/2016, 4:56:14 PM - info: Connecting to RabbitMQ
10/19/2016, 4:56:14 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 4:56:14 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 4:56:15 PM - info: In Uncaught exception: 
10/19/2016, 4:56:15 PM - error: Error: listen EADDRINUSE :::9010
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at Server._listen2 (net.js:1253:14)
    at listen (net.js:1289:10)
    at Server.listen (net.js:1385:5)
    at EventEmitter.listen (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/express/lib/application.js:617:24)
    at Object.<anonymous> (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/src/app.js:47:5)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Function.Module.runMain (module.js:575:10)
    at startup (node.js:160:18)
    at node.js:456:3
10/19/2016, 4:56:15 PM - info: Cleanup called
10/19/2016, 4:56:15 PM - info: MongoDB cleanup called
10/19/2016, 4:56:15 PM - info: Rabbitmq cleanup called
10/19/2016, 4:56:15 PM - error:  Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
10/19/2016, 4:56:39 PM - info: Connecting to MongoDB
10/19/2016, 4:56:39 PM - info: Connecting to RabbitMQ
10/19/2016, 4:56:39 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 4:56:39 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 4:56:39 PM - info: In Uncaught exception: 
10/19/2016, 4:56:39 PM - error: Error: listen EADDRINUSE :::9010
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at Server._listen2 (net.js:1253:14)
    at listen (net.js:1289:10)
    at Server.listen (net.js:1385:5)
    at EventEmitter.listen (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/express/lib/application.js:617:24)
    at Object.<anonymous> (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/src/app.js:47:5)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Function.Module.runMain (module.js:575:10)
    at startup (node.js:160:18)
    at node.js:456:3
10/19/2016, 4:56:39 PM - info: Cleanup called
10/19/2016, 4:56:39 PM - info: MongoDB cleanup called
10/19/2016, 4:56:39 PM - info: Rabbitmq cleanup called
10/19/2016, 4:56:39 PM - error:  Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
10/19/2016, 4:57:13 PM - info: Connecting to MongoDB
10/19/2016, 4:57:13 PM - info: Connecting to RabbitMQ
10/19/2016, 4:57:13 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 4:57:13 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 4:57:13 PM - info: In Uncaught exception: 
10/19/2016, 4:57:13 PM - error: Error: listen EADDRINUSE :::9010
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at Server._listen2 (net.js:1253:14)
    at listen (net.js:1289:10)
    at Server.listen (net.js:1385:5)
    at EventEmitter.listen (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/express/lib/application.js:617:24)
    at Object.<anonymous> (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/src/app.js:47:5)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Function.Module.runMain (module.js:575:10)
    at startup (node.js:160:18)
    at node.js:456:3
10/19/2016, 4:57:13 PM - info: Cleanup called
10/19/2016, 4:57:13 PM - info: MongoDB cleanup called
10/19/2016, 4:57:13 PM - info: Rabbitmq cleanup called
10/19/2016, 4:57:13 PM - error:  Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
10/19/2016, 4:58:10 PM - info: Connecting to MongoDB
10/19/2016, 4:58:10 PM - info: Connecting to RabbitMQ
10/19/2016, 4:58:10 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 4:58:10 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 4:58:10 PM - info: In Uncaught exception: 
10/19/2016, 4:58:10 PM - error: Error: listen EADDRINUSE :::9010
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at Server._listen2 (net.js:1253:14)
    at listen (net.js:1289:10)
    at Server.listen (net.js:1385:5)
    at EventEmitter.listen (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/express/lib/application.js:617:24)
    at Object.<anonymous> (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/src/app.js:47:5)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Function.Module.runMain (module.js:575:10)
    at startup (node.js:160:18)
    at node.js:456:3
10/19/2016, 4:58:10 PM - info: Cleanup called
10/19/2016, 4:58:10 PM - info: MongoDB cleanup called
10/19/2016, 4:58:10 PM - info: Rabbitmq cleanup called
10/19/2016, 4:58:10 PM - info: Process exiting: 1
10/19/2016, 4:58:29 PM - info: Connecting to MongoDB
10/19/2016, 4:58:29 PM - info: Connecting to RabbitMQ
10/19/2016, 4:58:29 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 4:58:29 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 4:58:29 PM - info: In Uncaught exception: 
10/19/2016, 4:58:29 PM - error: Error: listen EADDRINUSE :::9010
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at Server._listen2 (net.js:1253:14)
    at listen (net.js:1289:10)
    at Server.listen (net.js:1385:5)
    at EventEmitter.listen (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/express/lib/application.js:617:24)
    at Object.<anonymous> (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/src/app.js:47:5)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Function.Module.runMain (module.js:575:10)
    at startup (node.js:160:18)
    at node.js:456:3
10/19/2016, 4:58:29 PM - info: Cleanup called
10/19/2016, 4:58:29 PM - info: MongoDB cleanup called
10/19/2016, 4:58:29 PM - info: Rabbitmq cleanup called
10/19/2016, 4:58:29 PM - error:  Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
10/19/2016, 5:00:37 PM - info: Connecting to MongoDB
10/19/2016, 5:00:37 PM - info: Connecting to RabbitMQ
10/19/2016, 5:00:37 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 5:00:37 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 5:00:37 PM - info: Express server listening on port 9010
10/19/2016, 5:00:37 PM - error:  Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
10/19/2016, 5:00:37 PM - error: call attribute service in attribute service returned error for categoryId : 1000 { Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 9000 }
10/19/2016, 5:00:37 PM - info: MongoDB Connected!
10/19/2016, 5:00:38 PM - info: RabbitMQ connected
10/19/2016, 5:00:49 PM - info: getSeekerAttributesAndCategoryFromDb in jobseekerquestionsservice  called for jobSeekerId : 58074ee0e10743268790b228
10/19/2016, 5:00:49 PM - info: getAttributesForCategory in jobseekerquestionsservice  called for jobSeekerId : 58074ee0e10743268790b228
10/19/2016, 5:00:49 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 5:00:49 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 5:00:49 PM - error:  Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
10/19/2016, 5:00:49 PM - error: call attribute service in attribute service returned error for categoryId : 1000 { Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 9000 }
10/19/2016, 5:01:52 PM - info: Received SIGINT
10/19/2016, 5:01:52 PM - info: Cleanup called
10/19/2016, 5:01:52 PM - info: MongoDB cleanup called
10/19/2016, 5:01:52 PM - info: Rabbitmq cleanup called
10/19/2016, 5:01:52 PM - info: MongoDB connection closed
10/19/2016, 5:01:52 PM - info: RabbitMQ connection closed
10/19/2016, 5:01:55 PM - info: Connecting to MongoDB
10/19/2016, 5:01:55 PM - info: Connecting to RabbitMQ
10/19/2016, 5:01:55 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 5:01:55 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 5:01:55 PM - info: Express server listening on port 9010
10/19/2016, 5:01:55 PM - info: MongoDB Connected!
10/19/2016, 5:01:55 PM - info: RabbitMQ connected
10/19/2016, 5:01:59 PM - info: getIdArrayFromObjects in attribute service called
10/19/2016, 5:02:01 PM - info: getSeekerAttributesAndCategoryFromDb in jobseekerquestionsservice  called for jobSeekerId : 58074ee0e10743268790b228
10/19/2016, 5:02:01 PM - info: getAttributesForCategory in jobseekerquestionsservice  called for jobSeekerId : 58074ee0e10743268790b228
10/19/2016, 5:02:01 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 5:02:01 PM - info: getAttributeDifference in jobseekerquestionsservice  called 
10/19/2016, 5:02:01 PM - info: getQuestionsForAttributes in jobseekerquestionsservice  called 
10/19/2016, 5:02:01 PM - info: getQuestions in questionsservice  called 
10/19/2016, 5:02:01 PM - error:  statusCode=500, message=Something bad happened
10/19/2016, 5:02:38 PM - info: Received SIGINT
10/19/2016, 5:02:38 PM - info: Cleanup called
10/19/2016, 5:02:38 PM - info: MongoDB cleanup called
10/19/2016, 5:02:38 PM - info: Rabbitmq cleanup called
10/19/2016, 5:02:38 PM - info: MongoDB connection closed
10/19/2016, 5:02:38 PM - info: RabbitMQ connection closed
10/19/2016, 5:03:43 PM - info: Connecting to MongoDB
10/19/2016, 5:03:43 PM - info: Connecting to RabbitMQ
10/19/2016, 5:03:43 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 5:03:43 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 5:03:51 PM - info: Express server listening on port 9010
10/19/2016, 5:03:51 PM - error:  Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
10/19/2016, 5:03:51 PM - error: call attribute service in attribute service returned error for categoryId : 1000 { Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 9000 }
10/19/2016, 5:03:51 PM - info: MongoDB Connected!
10/19/2016, 5:03:51 PM - info: RabbitMQ connected
10/19/2016, 5:04:31 PM - info: getSeekerAttributesAndCategoryFromDb in jobseekerquestionsservice  called for jobSeekerId : 58074ee0e10743268790b228
10/19/2016, 5:04:33 PM - info: getAttributesForCategory in jobseekerquestionsservice  called for jobSeekerId : 58074ee0e10743268790b228
10/19/2016, 5:04:42 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 5:04:42 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 5:04:42 PM - error:  Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
10/19/2016, 5:04:42 PM - error: call attribute service in attribute service returned error for categoryId : 1000 { Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 9000 }
10/19/2016, 5:54:56 PM - info: getSeekerAttributesAndCategoryFromDb in jobseekerquestionsservice  called for jobSeekerId : 58074ee0e10743268790b228
10/19/2016, 5:55:06 PM - info: getAttributesForCategory in jobseekerquestionsservice  called for jobSeekerId : 58074ee0e10743268790b228
10/19/2016, 5:55:43 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 5:55:43 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 5:55:47 PM - info: In Uncaught exception: 
10/19/2016, 5:55:47 PM - error: Error: connection timeout
    at .<anonymous> (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/mongoose/lib/drivers/node-mongodb-native/connection.js:168:17)
    at emitTwo (events.js:106:13)
    at emit (events.js:191:7)
    at listener (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/mongodb/lib/db.js:1791:14)
    at emitOne (events.js:96:13)
    at emit (events.js:188:7)
    at Server.connect.errorHandler (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/mongodb/lib/server.js:270:14)
    at g (events.js:286:16)
    at emitOne (events.js:96:13)
    at emit (events.js:188:7)
    at eventHandler (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/mongodb-core/lib/topologies/server.js:322:12)
    at emitOne (events.js:96:13)
    at emit (events.js:188:7)
    at .<anonymous> (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/mongodb-core/lib/connection/pool.js:260:12)
    at g (events.js:286:16)
    at emitTwo (events.js:106:13)
10/19/2016, 5:55:47 PM - info: Cleanup called
10/19/2016, 5:55:47 PM - info: MongoDB cleanup called
10/19/2016, 5:55:47 PM - info: Rabbitmq cleanup called
10/19/2016, 5:55:47 PM - info: MongoDB connection closed
10/19/2016, 5:55:47 PM - error:  Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
10/19/2016, 5:55:47 PM - error: call attribute service in attribute service returned error for categoryId : 1000 { Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 9000 }
10/19/2016, 5:55:47 PM - info: RabbitMQ connection closed
10/19/2016, 5:56:23 PM - info: Connecting to MongoDB
10/19/2016, 5:56:23 PM - info: Connecting to RabbitMQ
10/19/2016, 5:56:23 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 5:56:23 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 5:56:32 PM - info: Express server listening on port 9010
10/19/2016, 5:56:32 PM - error:  Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
10/19/2016, 5:56:32 PM - error: call attribute service in attribute service returned error for categoryId : 1000 { Error: connect ECONNREFUSED 127.0.0.1:9000
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1080:14)
  code: 'ECONNREFUSED',
  errno: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 9000 }
10/19/2016, 5:56:32 PM - info: MongoDB Connected!
10/19/2016, 5:56:32 PM - info: RabbitMQ connected
10/19/2016, 5:56:58 PM - info: getSeekerAttributesAndCategoryFromDb in jobseekerquestionsservice  called for jobSeekerId : 58074ee0e10743268790b228
10/19/2016, 5:56:58 PM - info: getAttributesForCategory in jobseekerquestionsservice  called for jobSeekerId : 58074ee0e10743268790b228
10/19/2016, 6:00:25 PM - info: Connecting to MongoDB
10/19/2016, 6:00:25 PM - info: Connecting to RabbitMQ
10/19/2016, 6:00:29 PM - info: getBasicAttributes in attributeservice  called 
10/19/2016, 6:00:32 PM - info: callAttributeService in attribute service called for categoryId : 1000
10/19/2016, 6:00:41 PM - info: In Uncaught exception: 
10/19/2016, 6:00:41 PM - error: Error: listen EADDRINUSE :::9010
    at Object.exports._errnoException (util.js:1007:11)
    at exports._exceptionWithHostPort (util.js:1030:20)
    at Server._listen2 (net.js:1253:14)
    at listen (net.js:1289:10)
    at Server.listen (net.js:1385:5)
    at EventEmitter.listen (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/node_modules/express/lib/application.js:617:24)
    at Object.<anonymous> (/Users/shubhammakwana/Documents/Workspace/job-seeker-api/src/app.js:47:5)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Timeout.Module.runMain [as _onTimeout] (module.js:575:10)
    at tryOnTimeout (timers.js:224:11)
    at Timer.listOnTimeout (timers.js:198:5)
10/19/2016, 6:00:41 PM - info: Cleanup called
10/19/2016, 6:00:41 PM - info: MongoDB cleanup called
10/19/2016, 6:00:41 PM - info: Rabbitmq cleanup called
10/19/2016, 6:00:41 PM - info: Process exiting: 1
